### ISSUE TITLE
Easier and faster to just hold shift and scroll down?

### ISSUE THREAD

**Issue by y3sh** [mrishab/google-photos-delete-tool#38]
Good script... tried it, then realized it's actually not bad to delete thousands of photos anymore via the UI... maybe they've pushed some UI updates to improve this UX without actual "delete all"? 

1. Click the checkmark on the first image
2. Hold shift
3. Scroll down about 500 or so pics (not too much though or this won't work)
4. Let the UI finish loading for a couple seconds
5. Still holding shift, notice on hover that checkmarks shade-in for all photos between 1 and wherever you're at in the scroll
6. Click on a photo, hold shift again and scroll down some more
7. Hit delete

I just deleted 23,000 photos in about 3 minutes. Might be worth a try 😄

**Comment by dinozarafonitis** [comment]
@ShayanYaseen you are right, eventually I did the same thing. Initially I tried the script but it was hanging so I had to re-load it, and it was also slow (even with the images being blocked from loading)

It was faster to manually delete them by zooming-out the browser and the make sure I block the images from loading in the UI while on `photos.google.com`

![image](https://user-images.githubusercontent.com/29850103/132492195-9e444591-8d1c-4b27-b9ca-b4e773652f43.png)

**Comment by macsir** [comment]
The manual operation doesn't work well if you have years' worth of photos there.

**Comment by AppleJack0012** [comment]
I have 1.9m+ photos. I am not going to manually delete them. Most people who came for this script has thousands and many years worth and does not have the time to hold shift and scroll down.
